0800-appendix-58067-gramophone.Rmd

```diff
@@ -247,7 +259,7 @@ and steelhead.   The crossings location adjacent to the community of
 Witset, BC is ideal for community involvement in the project.  Additionally, because `r fpr::fpr_my_pscis_info(col_pull = road_name)` is a non-paved road with a crossing location that can be accessed from either side, the costs of the work are low when compared to paved road and highway options. Crossing
 `r as.character(my_site)` was ranked as a
 `r fpr::fpr_my_priority_info(col_pull = priority)` priority for
-proceeding to design for replacement.  Elecrofishing results at the site indicated that the stream is productive in terms of rainbow trout however the lack of coho presence was surprising.  It is recommended that the stream be surveyed all the way down to the Bulkley River to confirm that there are no barriers to migration downstream of the crossing before remedial works are planned.
+proceeding to design for replacement.  Electrofishing results at the site indicated that the stream is productive in terms of rainbow trout however after the 2023 re-assessment, it is presumed the falls and de-watering located downstream are likely a barrier to Coho.
```

Comment:
rock outcrops and chute within the canyon located at approximately 500m from the confluence with the Bulkley River likely `presented` (tense) a barrier to upstream migration of coho in 2022 and 2023

plus the dewatering.  I will give it a shot now